packages/jest-types/src/Circus.ts

```diff
@@ -16,7 +16,7 @@ export type BlockMode = void | 'skip' | 'only' | 'todo';
 export type TestMode = BlockMode;
 export type TestName = Global.TestName;
 export type TestFn = Global.TestFn;
-export type HookFn = (done?: DoneFn) => Promise<any> | null | undefined;
```

Comment:
we don't actually accept `null`, we check for `void 0` which is an non-strict mode way of saying `undefined`